assets/content/cookbook/Beginner/string-variable-reflection.md

```diff
@@ -0,0 +1,38 @@
+[tags]: / "beginner,other"
```

Comment:
"Beginner" and "other" are categories, not tags. They are mutually exclusive. I propose using "reflection" for the tag.